docs/painless/painless-contexts/painless-reindex-context.asciidoc

```diff
@@ -0,0 +1,68 @@
+[[painless-reindex-context]]
+=== Reindex Context
+
+Use a Painless script in a {es_version}/docs-reindex.html[reindex] operation to
+add, modify, or delete fields within each document in an original index as its
+re-indexed into a target index.
+
```

Comment:
"it's reindexed"  We don't hyphenate reindex--we tend to close words up instead of hyphenating in general.

Reply:
Fixed.